packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.container.tsx

```diff
@@ -85,9 +93,36 @@ S extends MyAccountMyWishlistContainerState = MyAccountMyWishlistContainerState,
             isLoading: false,
             loadingItemsMap: {} as Record<string, boolean>,
             isQtyUpdateInProgress: false,
+            productsPerPage: 10,
         } as S;
     }
 
+    componentDidMount(): void {
+        const { updateWishlistProducts, location } = this.props;
+        const { productsPerPage } = this.state;
+        const page = getPageFromUrl(location);
+
+        updateWishlistProducts(page, productsPerPage);
+    }
```

Comment:
Better to move this to constructor. Is products per page static for entering first-time wishlist or it can be configurated through BE? (as it is done for product listing on PLP)